packages/blade/tsconfig-generate-types.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
`tsconfig` which just focusses on generating types. It extends the base config itself. Kept it separate instead of passing the command line arguments for enabling/disabling flags for generating types

Reply:
Can we name this file `tsconfig-generate-custom-types`? Because it's only responsible for custom types & not all types.

Reply:
This is actually responsible for generating all the typescript types. This is the config that actually emits types for all of our TS files